faf-commons-api/src/main/java/com/faforever/commons/api/dto/NeroxisGeneratorParams.java

```diff
@@ -0,0 +1,16 @@
+package com.faforever.commons.api.dto;
+
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@EqualsAndHashCode(callSuper = true)
+@Getter
+@Setter
```

Comment:
Use a value class instead? (Maybe I'm missing a potential compile error, but I see no reason for that)

Reply:
You need a no args constructor for the json I believe

Reply:
No, I think that was only the case in older Jackson versions.

Reply:
I tried using value yesterday and in the moderator client I got the error during runtime that it couldn't parse it due to the lack of a default constructor.

Reply:
As discussed adding 
lombok.anyConstructor.addConstructorProperties = true

to lombok.config will make @Value classes work with Jackson.

Reply:
yep just forgot to push it. should be changed now